pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -88,14 +96,70 @@ func TestTargetConfigController(t *testing.T) {
 				u.FakeEtcdMemberWithoutServer(0),
 				u.FakeEtcdMemberWithoutServer(2),
 			},
+			etcdBackupSpec:    nil,
 			etcdMembersEnvVar: "1,3",
 		},
+		{
+			name: "BackupVar Test HappyPath",
+			objects: []runtime.Object{
+				u.BootstrapConfigMap(u.WithBootstrapStatus("complete")),
+			},
+			staticPodStatus: u.StaticPodOperatorStatus(
+				u.WithLatestRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+			),
+			etcdMembers: []*etcdserverpb.Member{
+				u.FakeEtcdMemberWithoutServer(0),
+				u.FakeEtcdMemberWithoutServer(1),
+				u.FakeEtcdMemberWithoutServer(2),
+			},
+			etcdBackupSpec:    u.CreateEtcdBackupSpecPtr("GMT", "0 */2 * * *"),
+			etcdMembersEnvVar: "1,2,3",
+		},
+		{
+			name: "Backup Var Test with empty spec",
+			objects: []runtime.Object{
+				u.BootstrapConfigMap(u.WithBootstrapStatus("complete")),
+			},
+			staticPodStatus: u.StaticPodOperatorStatus(
+				u.WithLatestRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+			),
+			etcdMembers: []*etcdserverpb.Member{
+				u.FakeEtcdMemberWithoutServer(0),
+				u.FakeEtcdMemberWithoutServer(1),
+				u.FakeEtcdMemberWithoutServer(2),
+			},
+			etcdBackupSpec:    nil,
+			etcdMembersEnvVar: "1,2,3",
+		},
 	}
 	for _, scenario := range scenarios {
 		t.Run(scenario.name, func(t *testing.T) {
-			eventRecorder, _, controller := getController(t, scenario.staticPodStatus, scenario.objects, scenario.etcdMembers)
+			backupVar := backuphelpers.NewDisabledBackupConfig()
+			eventRecorder, _, controller, fakeKubeClient := getController(t, scenario.staticPodStatus, scenario.objects, scenario.etcdMembers, backupVar)
+			backupVar.SetBackupSpec(scenario.etcdBackupSpec)
 			err := controller.sync(context.TODO(), factory.NewSyncContext("test", eventRecorder))
 			assert.Equal(t, scenario.expectedErr, err)
+
+			if scenario.expectedErr != nil {
+				return
+			}
+
+			etcdPodCM, err := fakeKubeClient.CoreV1().ConfigMaps(operatorclient.TargetNamespace).Get(context.TODO(), "etcd-pod", metav1.GetOptions{})
+			require.NoError(t, err)
+			var expStr string
+
+			if scenario.etcdBackupSpec != nil {
+				expStr = "    args:\n    - --enabled=true\n    - --timezone=GMT\n    - --schedule=0 */2 * * *"
+			} else {
+				expStr = "    args:\n    - --enabled=false"
+			}
+			require.Contains(t, etcdPodCM.Data["pod.yaml"], expStr)
```

Comment:
```suggestion

    expStr := " args:\n - --enabled=false"
    if scenario.etcdBackupSpec != nil {
       expStr = " args:\n - --enabled=true\n - --timezone=GMT\n - --schedule=0 */2 * * *"
    }
    

   if _, ok := etcdPodCM.Data["pod.yaml"]; ok {
       require.Contains(t, etcdPodCM.Data["pod.yaml"], expStr)
   }
			
```

Reply:
lgtm 👍🏽

Reply:
sorry, the expectedErr early return also does the same job

```
			if scenario.expectedErr != nil {
				return
			}
```

Reply:
mir egal :D

Reply:
mir auch, let's leave it the way it is

Reply:
👍🏽